time-macros/src/lib.rs

```diff
@@ -89,3 +91,114 @@ pub fn format_description(input: TokenStream) -> TokenStream {
         DESCRIPTION
     }}
 }
+
+fn make_serde_serializer_module(
+    mod_name: proc_macro::Ident,
+    items: impl to_tokens::ToTokens,
+    formattable: TokenStream,
+) -> TokenStream {
+    let serialize_fns = quote! {
+        pub fn serialize<S: ::serde::Serializer>(
+                datetime: &#(formattable.clone()),
```

Comment:
Check the formatting on code inside `quote!` — rustfmt doesn't handle it.

Reply:
I think I got it now